### ISSUE TITLE
Can this be used in production code ? 

### ISSUE THREAD

**Issue by kartavya-ramnani** [bufbuild/buf#22]
Hi, 
I am going through buf documentation and I love and I wish to use it in my production code.

However, I have noticed that this is early days for buf and it is still under development. 

Do you recommend buf to be used in production code as of this moment?

**Comment by bufdev** [comment]
Buf is still in pre-release, but the core feature set currently exposed will almost for sure remain. During the beta period, there might be changes such as config file structure, lint groups, command names, etc - all the features will likely still be exposed however. Any such changes will be clearly labeled as breaking in the release notes, along with the transition process.

Before v1.0, as long as you stay locked to a minor release point, you will be fine - no such changes will happen on a micro release, only bug fixes.

In short: fine to use in prod (right now, Buf is just a CLI tool that provides building, linting, and breaking change detection, so it's a build-time tool anyways), as long as you stay locked to a minor, and understand you may have to make minor changes to your config files, command names, etc as you upgrade.
